src/xrc/src/api.rs

```diff
@@ -235,17 +236,76 @@ fn validate(rate: QueriedExchangeRate) -> Result<QueriedExchangeRate, ExchangeRa
     }
 }
 
+#[derive(Debug, PartialEq, Eq)]
+enum NormalizedTimestampType {
+    /// The timestamp is within the past minute.
+    RequestedOrCurrent,
+    /// The timestamp is within 1-2 minutes ago.
```

Comment:
Why does `Past` mean "the timestamp is within 1-2 minutes ago"?

Reply:
Having read the code below, "within 1-2 minutes ago" refers to how we actually use this type.
I think the comment is confusing here. It should simply say that it refers to a timestamp in the past.

Reply:
That makes sense. Updated.